packages/typescript/index.js

```diff
@@ -51,6 +51,15 @@ module.exports = {
             '@typescript-eslint/restrict-template-expressions': 'error',
             '@typescript-eslint/unbound-method': 'error',
           },
+        }, {
+          // https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/unbound-method.md
+          files: ['**/__tests__/**/*.ts', '**/*.spec.ts', '**/*.test.ts'],
```

Comment:
Check https://jestjs.io/docs/configuration#testmatch-arraystring and https://eslint.org/docs/latest/user-guide/configuring/configuration-files-new#specifying-files-and-ignores